packages/riipen-ui/src/components/Link.test.jsx

```diff
@@ -0,0 +1,178 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Link from "./Link.jsx";
+
+describe("<Link>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(
+        <Link href="google.com">
+          <span>link</span>
+        </Link>
+      );
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(
+      <Link href="google.com">
+        <span>link</span>
+      </Link>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(
+        <Link href="google.com">
+          <span>link</span>
+        </Link>
+      );
+
+      const component = wrapper.find("Link");
+      expect(component.props().classes).toEqual([]);
+      expect(component.props().color).toEqual("primary");
+      expect(component.props().component).toEqual("a");
+      expect(component.props().underline).toEqual("none");
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders given children", () => {
+      const child = <span>link</span>;
+
+      const wrapper = mount(<Link href="google.com">{child}</Link>);
+
+      expect(wrapper.childAt(0).contains(child)).toEqual(true);
```

Comment:
Can this test use find("Link") instead of childAt(0) to make it a bit easier to read? (and similar to the other tests)